app/src/main/java/com/jnj/vaccinetracker/common/util/FileUtil.kt

```diff
@@ -55,11 +63,18 @@ class FileUtil {
                 } else {
                     Toast.makeText(context, context.getString(R.string.visits_overview_no_available_app_message), Toast.LENGTH_LONG).show()
                 }
-            } catch (e: Exception) {
-                Log.e("OpenFile", "Failed to open file", e)
+            } else {
+                Log.e("OpenFile", "File does not exist: ${file.absolutePath}")
                 Toast.makeText(context, context.getString(R.string.visits_overview_failed_to_open_file_message), Toast.LENGTH_LONG).show()
             }
         }
-    }
 
-}
\ No newline at end of file
+        private fun getMimeType(fileName: String): String? {
```

Comment:
this method always returns String, so return type can be String instead of String?